onnxruntime/core/providers/rocm/rocm_kernel.h

```diff
@@ -58,14 +60,23 @@ class RocmKernel : public OpKernel {
     return provider_->AllocateBufferOnCPUPinned<T>(count_or_bytes);
   }
 
-  inline void AddDeferredReleaseCPUPtr(void* p) const {
+  inline void AddDeferredReleaseCPUPtr(void* p, onnxruntime::Stream* /*TODO*/= nullptr) const {
```

Comment:
why you add this? have you checked does this work with the old "deferred release CPU ptr" approach?